libraries/chain/asset_evaluator.cpp

```diff
@@ -56,7 +56,8 @@ void_result asset_create_evaluator::do_evaluate( const asset_create_operation& o
    FC_ASSERT( asset_symbol_itr == asset_indx.end() );
 
    // This must remain due to "BOND.CNY" being allowed before this HF
-   if( d.head_block_time() > HARDFORK_385_TIME )
+   const time_point_sec now = d.head_block_time();
```

Comment:
add comment:
`// define now from current block time once for determinism`
move comment one line down above if statement:
`// This must remain due to "BOND.CNY" being allowed before this HF`
a few lines down, the comment
`// Taker fees should be zero until activation of BSIP81`
can also move above its respective if statement